templates/shared/_polyfill.html

```diff
@@ -0,0 +1,7 @@
+    <script>
```

Comment:
Can you add a comment here to state it is the js polyfills

Reply:
Done